src/main/java/it/gov/pagopa/arc/connector/bizevents/paidnotice/BizEventsPaidNoticeConnectorImpl.java

```diff
@@ -22,65 +25,79 @@ public class BizEventsPaidNoticeConnectorImpl implements BizEventsPaidNoticeConn
     private static final String ERROR_MESSAGE_INVOCATION_EXCEPTION = "An error occurred handling request from biz-Events";
     private final String apikey;
     private final BizEventsPaidNoticeRestClient bizEventsPaidNoticeRestClient;
+    private final BizEventsPaidNoticeDTO2NoticesListResponseDTOMapper bizEventsPaidNoticeDTO2NoticesListResponseDTOMapper;
     private final ObjectMapper objectMapper;
 
     public BizEventsPaidNoticeConnectorImpl(@Value("${rest-client.biz-events.paid-notice.api-key}") String apikey,
-                                            BizEventsPaidNoticeRestClient bizEventsPaidNoticeRestClient, ObjectMapper objectMapper) {
+                                            BizEventsPaidNoticeRestClient bizEventsPaidNoticeRestClient, BizEventsPaidNoticeDTO2NoticesListResponseDTOMapper bizEventsPaidNoticeDTO2NoticesListResponseDTOMapper, ObjectMapper objectMapper) {
         this.apikey = apikey;
         this.bizEventsPaidNoticeRestClient = bizEventsPaidNoticeRestClient;
+        this.bizEventsPaidNoticeDTO2NoticesListResponseDTOMapper = bizEventsPaidNoticeDTO2NoticesListResponseDTOMapper;
         this.objectMapper = objectMapper;
     }
 
     @Override
-    public BizEventsPaidResponseDTO getPaidNoticeList(String fiscalCode, String continuationToken, Integer size, Boolean isPayer, Boolean isDebtor, String orderBy, String ordering) {
-        Response response;
-        BizEventsPaidResponseDTO bizEventsPaidResponseDTO;
+    public NoticesListResponseDTO getPaidNoticeList(String fiscalCode, NoticeRequestDTO noticeRequestDTO) {
+        String continuationToken = null;
+        Integer size = null;
+        Boolean isPayer = null;
+        Boolean isDebtor = null;
+        String orderBy = null;
+        String ordering = null;
+
+        if(noticeRequestDTO != null){
+            continuationToken = noticeRequestDTO.getContinuationToken();
+            size = noticeRequestDTO.getSize();
+            isPayer = noticeRequestDTO.getPaidByMe();
+            isDebtor = noticeRequestDTO.getRegisteredToMe();
+            orderBy = noticeRequestDTO.getOrderBy();
+            ordering = noticeRequestDTO.getOrdering();
+        }
```

Comment:
If this parameter cannot be null (otherwise it's a bug), you could not handle the null value
This restriction would simplify this method (you are not more requested to handle and test this case!)